terraform/pagerduty/member-services-integrations.tf

```diff
@@ -2042,3 +2042,37 @@ resource "pagerduty_event_orchestration_service" "corporate-staff-rostering-prep
     actions {}
   }
 }
+
+resource "pagerduty_service" "sprinkler-development" {
+  name                    = "sptinkler-development"
+  description             = "sprinkler-development"
+  auto_resolve_timeout    = 345600
+  acknowledgement_timeout = "null"
+  escalation_policy       = pagerduty_escalation_policy.member_policy.id
+  alert_creation          = "create_alerts_and_incidents"
+}
+
+resource "pagerduty_service_integration" "sprinkler-integration" {
```

Comment:
Do we need to include commented-out sections? Probably not.

Reply:
I'll remove the commented out bits.